pkg/util/util.go

```diff
@@ -64,6 +64,11 @@ func IsRunModeLocal() bool {
 
 // GetOperatorNamespace returns the namespace the operator should be running in.
 var GetOperatorNamespace = func(logger logr.Logger) (string, error) {
+	ns, err := GetWatchNamespace()
```

Comment:
OperatorNamespace and WatchNamespace are two distinct values, please avoid this.